contracts/dapis/DapiServer.sol

```diff
@@ -833,20 +838,25 @@ contract DapiServer is
     /// offset and scale.
     /// @param initialValue Initial value
     /// @param updatedValue Updated value
+    /// @param deviationReference Reference value that deviation will be
+    /// calculated against
```

Comment:
Should we re-word the @dev text above and say that in those use the `deviationReference` parameter?

Reply:
Good catch